module/VuFind/src/VuFind/View/Helper/Root/Record.php

```diff
@@ -668,7 +668,8 @@ public function getLinkDetails($openUrlActive = false)
             return $link;
         };
 
-        return array_map($formatLink, $urls);
+        $arr = array_map($formatLink, $urls);
+        return array_values(array_unique($arr, SORT_REGULAR));
```

Comment:
Maybe it's worth wrapping `array_values(array_unique($arr, SORT_REGULAR));` into a `protected function deduplicateUrls($urls)` definition (similar to the strategy of the earlier PR) so that if people want to apply a different algorithm, there's an easy extension point for them to utilize.